Source/WebCore/rendering/Grid.cpp

```diff
@@ -36,19 +37,19 @@ Grid::Grid(RenderGrid& grid)
 {
 }
 
-unsigned Grid::numTracks(GridTrackSizingDirection direction) const
+unsigned Grid::numTracks(Style::GridTrackSizingDirection direction) const
 {
-    if (direction == GridTrackSizingDirection::ForRows)
+    if (direction == Style::GridTrackSizingDirection::Rows)
         return m_grid.size();
     return m_grid.size() ? m_grid[0].size() : 0;
 }
 
 void Grid::ensureGridSize(unsigned maximumRowSize, unsigned maximumColumnSize)
 {
-    ASSERT(static_cast<int>(maximumRowSize) < GridPosition::max() * 2);
-    ASSERT(static_cast<int>(maximumColumnSize) < GridPosition::max() * 2);
-    const size_t oldColumnSize = numTracks(GridTrackSizingDirection::ForColumns);
-    const size_t oldRowSize = numTracks(GridTrackSizingDirection::ForRows);
+    ASSERT(static_cast<int>(maximumRowSize) < Style::GridPosition::max() * 2);
+    ASSERT(static_cast<int>(maximumColumnSize) < Style::GridPosition::max() * 2);
+    const size_t oldColumnSize = numTracks(Style::GridTrackSizingDirection::Columns);
+    const size_t oldRowSize = numTracks(Style::GridTrackSizingDirection::Rows);
```

Comment:
Not new to this patch: We typically don’t use `const` for local variable scalars in WebKit.
```suggestion
    size_t oldColumnSize = numTracks(Style::GridTrackSizingDirection::Columns);
    size_t oldRowSize = numTracks(Style::GridTrackSizingDirection::Rows);
```